Tools/Scripts/webkitpy/common/checkout/checkout.py

```diff
@@ -45,6 +46,27 @@
 # FIXME: Move a bunch of ChangeLog-specific processing from SCM to this object.
 # NOTE: All paths returned from this class should be absolute.
 class Checkout(object):
+    COMMIT_SUBJECT_RE = re.compile(rb'Subject: \[PATCH ?(\d+\/\d+)?] (.+)')
+    FILTER_BRANCH_PROGRAM = '''import re
+import sys
+
+lines = [l for l in sys.stdin]
+for s in re.split(r' (Need the bug URL \(OOPS!\).)|(\S+:\/\/\S+)', lines[0].rstrip()):
+    if s and s != ' ':
+        print(s)
+for l in lines[1:]:
+    sys.stdout.write(l)
+'''
```

Comment:
To add some context here: `git am` will squash all lines before the first newline into a single line. If we attempt to put this code in `filter_patch_content`, `git am` will actually _ignore_ all lines between the first line and the first empty line. `git filter-branch` after apply the patch is the only work-around.